### ISSUE TITLE
Release 2021.10 - RC1

### ISSUE THREAD

**Issue by benpicco** [RIOT-OS/Release-Specs#235]
This issue lists the status of all tests for the [Release Candidate 1 of the 2021.10](https://github.com/RIOT-OS/RIOT/releases/tag/2021.10-RC1) release.

Specs tested:
- [ ] [01-ci](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/01-ci/README.md)
  - [ ] [Task #01 - Compile test](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/01-ci/README.md#task-01---compile-test)
  - [x] [Task #02 - Unittests on native](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/01-ci/README.md#task-02---unittests-on-native) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #03 - Unittests on native separated](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/01-ci/README.md#task-03---unittests-on-native-separated) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #04 - Unittests on iotlab-m3](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/01-ci/README.md#task-04---unittests-on-iotlab-m3) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
- [ ] [02-tests](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/02-tests/README.md)
  - [ ] [Task #01 - All tests on native](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/02-tests/README.md#task-01---all-tests-on-native)
  - [ ] [Task #02 - Subset of tests on iotlab-m3](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/02-tests/README.md#task-02---subset-of-tests-on-iotlab-m3)
  - [ ] [Task #03 - Subset of tests on samr21-xpro](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/02-tests/README.md#task-03---subset-of-tests-on-samr21-xpro)
- [x] [03-single-hop-ipv6-icmp](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/03-single-hop-ipv6-icmp/README.md)
  - [x] [Task #01 - ICMPv6 multicast echo on native](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/03-single-hop-ipv6-icmp/README.md#task-01---icmpv6-multicast-echo-on-native) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #02 - ICMPv6 link-local echo on native](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/03-single-hop-ipv6-icmp/README.md#task-02---icmpv6-link-local-echo-on-native) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #03 - ICMPv6 link-local echo on native (1 hour)](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/03-single-hop-ipv6-icmp/README.md#task-03---icmpv6-link-local-echo-on-native-1-hour) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #04 - ICMPv6 stress test on native (1 hour)](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/03-single-hop-ipv6-icmp/README.md#task-04---icmpv6-stress-test-on-native-1-hour) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #05 - ICMPv6 stress test on native (neighbor cache stress)](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/03-single-hop-ipv6-icmp/README.md#task-05---icmpv6-stress-test-on-native-neighbor-cache-stress) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #06 - ICMPv6 link-local echo on native (IPv6 fragmentation)](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/03-single-hop-ipv6-icmp/README.md#task-06---icmpv6-link-local-echo-on-native-ipv6-fragmentation) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
- [ ] [04-single-hop-6lowpan-icmp](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/04-single-hop-6lowpan-icmp/README.md)
  - [x] [Task #01 - ICMPv6 link-local echo with iotlab-m3](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/04-single-hop-6lowpan-icmp/README.md#task-01---icmpv6-link-local-echo-with-iotlab-m3) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #02 - ICMPv6 multicast echo with iotlab-m3/samr21-xpro](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/04-single-hop-6lowpan-icmp/README.md#task-02---icmpv6-multicast-echo-with-iotlab-m3samr21-xpro) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #03 - ICMPv6 echo with large payload](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/04-single-hop-6lowpan-icmp/README.md#task-03---icmpv6-echo-with-large-payload) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #04 - ICMPv6 echo with iotlab-m3/samr21-xpro 15 minutes](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/04-single-hop-6lowpan-icmp/README.md#task-04---icmpv6-echo-with-iotlab-m3samr21-xpro-15-minutes) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [ ] [Task #05 (Experimental) - ICMPv6 multicast echo with samr21-xpro/cc2538](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/04-single-hop-6lowpan-icmp/README.md#task-05-experimental---icmpv6-multicast-echo-with-samr21-xprocc2538)
  - [ ] [Task #06 (Experimental)- ICMPv6 link-local echo with samr21-xpro/cc2538](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/04-single-hop-6lowpan-icmp/README.md#task-06-experimental---icmpv6-link-local-echo-with-samr21-xprocc2538)
  - [ ] [Task #07 (Experimental) - ICMPv6 multicast echo with samr21-xpro/zero + xbee](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/04-single-hop-6lowpan-icmp/README.md#task-07-experimental---icmpv6-multicast-echo-with-samr21-xprozero--xbee)
  - [ ] [Task #08 (Experimental) - ICMPv6 echo with samr21-xpro/zero + xbee](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/04-single-hop-6lowpan-icmp/README.md#task-08-experimental---icmpv6-echo-with-samr21-xprozero--xbee)
  - [x] [Task #09 - ICMPv6 stress test on iotlab-m3](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/04-single-hop-6lowpan-icmp/README.md#task-09---icmpv6-stress-test-on-iotlab-m3) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [ ] [Task #10 (Experimental) - ICMPv6 echo with large payload (IPv6 fragmentation)](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/04-single-hop-6lowpan-icmp/README.md#task-10-experimental---icmpv6-echo-with-large-payload-ipv6-fragmentation)
  - [x] [Task #11 (Experimental) - ICMPv6 stress test on nrf802154](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/04-single-hop-6lowpan-icmp/README.md#task-11-experimental---icmpv6-stress-test-on-nrf802154) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #12 (Experimental) - ICMPv6 multicast echo with iotlab-m3/nrf802154](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/04-single-hop-6lowpan-icmp/README.md#task-12-experimental---icmpv6-multicast-echo-with-iotlab-m3nrf802154) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #13 (Experimental) - ICMPv6 link-local echo with iotlab-m3/nrf802154](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/04-single-hop-6lowpan-icmp/README.md#task-13-experimental---icmpv6-link-local-echo-with-iotlab-m3nrf802154) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
- [ ] [05-single-hop-route](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/05-single-hop-route/README.md)
  - [x] [Task #01 - ICMPv6 echo unicast addresess on native (default route)](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/05-single-hop-route/README.md#task-01---icmpv6-echo-unicast-addresess-on-native-default-route) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #02 - ICMPv6 echo unicast addresess on iotlab-m3 (default route)](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/05-single-hop-route/README.md#task-02---icmpv6-echo-unicast-addresess-on-iotlab-m3-default-route) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #03 - ICMPv6 echo unicast addresess on native (specific route)](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/05-single-hop-route/README.md#task-03---icmpv6-echo-unicast-addresess-on-native-specific-route) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #04 - ICMPv6 echo unicast addresess on iotlab-m3 (static route)](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/05-single-hop-route/README.md#task-04---icmpv6-echo-unicast-addresess-on-iotlab-m3-static-route) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
- [ ] [06-single-hop-udp](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/06-single-hop-udp/README.md)
  - [x] [Task #01 - UDP on iotlab-m3](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/06-single-hop-udp/README.md#task-01---udp-on-iotlab-m3) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #02 - UDP on iotlab-m3 (UDP port compression)](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/06-single-hop-udp/README.md#task-02---udp-on-iotlab-m3-udp-port-compression) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #03 - UDP on native (non-existent neighbor)](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/06-single-hop-udp/README.md#task-03---udp-on-native-non-existent-neighbor) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #04 - UDP on iotlab-m3 (non-existent neighbor)](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/06-single-hop-udp/README.md#task-04---udp-on-iotlab-m3-non-existent-neighbor) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #05 - Empty UDP on native](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/06-single-hop-udp/README.md#task-05---empty-udp-on-native) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #06 - Empty UDP on iotlab-m3](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/06-single-hop-udp/README.md#task-06---empty-udp-on-iotlab-m3) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
- [ ] [07-multi-hop](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/07-multi-hop/README.md)
  - [x] [Task #01 - ICMPv6 echo on iotlab-m3 with three hops (static route)](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/07-multi-hop/README.md#task-01---icmpv6-echo-on-iotlab-m3-with-three-hops-static-route) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #02 - UDP on iotlab-m3 with three hops (static route)](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/07-multi-hop/README.md#task-02---udp-on-iotlab-m3-with-three-hops-static-route) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #03 - ICMPv6 echo on iotlab-m3 with three hops (RPL route)](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/07-multi-hop/README.md#task-03---icmpv6-echo-on-iotlab-m3-with-three-hops-rpl-route) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #04 - UDP on iotlab-m3 with three hops (RPL route)](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/07-multi-hop/README.md#task-04---udp-on-iotlab-m3-with-three-hops-rpl-route) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [ ] [Task #05 (Experimental) - UDP with large payload on iotlab-m3 with three hops (RPL route)](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/07-multi-hop/README.md#task-05-experimental---udp-with-large-payload-on-iotlab-m3-with-three-hops-rpl-route)
- [ ] [08-interop](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/08-interop/README.md)
  - [x] [Task #01 - ICMPv6 echo between native and Linux](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/08-interop/README.md#task-01---icmpv6-echo-between-native-and-linux) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #02 - ICMPv6 echo between iotlab-m3 and Linux with 6LowPAN](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/08-interop/README.md#task-02---icmpv6-echo-between-iotlab-m3-and-linux-with-6lowpan) @fjmolinas (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-934316873)
  - [ ] [Task #03 - UDP exchange between iotlab-m3 and Contiki](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/08-interop/README.md#task-03---udp-exchange-between-iotlab-m3-and-contiki)
  - [ ] [Task #04 - ICMPv6 echo between iotlab-m3 and Internet host through Linux with 6LowPAN](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/08-interop/README.md#task-04---icmpv6-echo-between-iotlab-m3-and-internet-host-through-linux-with-6lowpan)
  - [x] [Task #05 - ICMPv6 echo between iotlab-m3 and Internet host through RIOT border router](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/08-interop/README.md#task-05---icmpv6-echo-between-iotlab-m3-and-internet-host-through-riot-border-router) @fjmolinas (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-934226122) 
  - [x] [Task #06 - UDP between iotlab-m3 and Internet host through RIOT border router](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/08-interop/README.md#task-06---udp-between-iotlab-m3-and-internet-host-through-riot-border-router) @fjmolinas (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-934249288) 
  - [x] [Task #07 - UDP between iotlab-m3 and Internet host through RIOT border router (200b payload)](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/08-interop/README.md#task-07---udp-between-iotlab-m3-and-internet-host-through-riot-border-router-200b-payload) @fjmolinas (https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-934251135) 
  - [x] [Task #08 - UDP between GNRC and lwIP on iotlab-m3](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/08-interop/README.md#task-08---udp-between-gnrc-and-lwip-on-iotlab-m3) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] ~~Task #09 - deprecated~~
  - [x] ~~Task #10 - deprecated~~
  - [ ] [Task #11 - UDP exchange between iotlab-m3 and Zephyr](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/08-interop/README.md#task-11---udp-exchange-between-iotlab-m3-and-zephyr) @leandrolanzieri (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-934256406)
  - [ ] [Task #12 - ICMPv6 echo between Border Router with WiFi uplink to named Internet host](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/08-interop/README.md#task-12---icmpv6-echo-between-border-router-with-wifi-uplink-to-named-internet-host)
  - [ ] [Task #13 - ICMPv6 echo between ESP and named Internet host through Linux with 6LowPAN](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/08-interop/README.md#task-13---icmpv6-echo-between-esp-and-named-internet-host-through-linux-with-6lowpan)
- [x] [09-coap](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/09-coap/README.md)
  - [x] [Task #01 - CORD Endpoint](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/09-coap/README.md#task-01---cord-endpoint) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #02 - Confirmable retries](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/09-coap/README.md#task-02---confirmable-retries) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #03 - Block1](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/09-coap/README.md#task-03---block1) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #04 - Block2](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/09-coap/README.md#task-04---block2) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #05 - Observe registration and notification](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/09-coap/README.md#task-05---observe-registration-and-notification) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
- [x] [10-icmpv6-error](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/10-icmpv6-error/README.md)
  - [x] [Task #01 - Destination unreachable - no route to destination](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/10-icmpv6-error/README.md#task-01---destination-unreachable---no-route-to-destination) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #02 - Destination unreachable - Beyond scope of source address](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/10-icmpv6-error/README.md#task-02---destination-unreachable---beyond-scope-of-source-address) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #03 - Destination unreachable - address unreachable (target node address)](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/10-icmpv6-error/README.md#task-03---destination-unreachable---address-unreachable-target-node-address) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #04 - Destination unreachable - address unreachable (neighbor cache miss)](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/10-icmpv6-error/README.md#task-04---destination-unreachable---address-unreachable-neighbor-cache-miss) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #05 - Destination unreachable - port unreachable](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/10-icmpv6-error/README.md#task-05---destination-unreachable---port-unreachable) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #06 - Destination unreachable - port unreachable (large payload)](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/10-icmpv6-error/README.md#task-06---destination-unreachable---port-unreachable-large-payload) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #07 - Packet too big](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/10-icmpv6-error/README.md#task-07---packet-too-big) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #08 - Time exceeded - hop limit exceeded in transit](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/10-icmpv6-error/README.md#task-08---time-exceeded---hop-limit-exceeded-in-transit) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #09 - Parameter problem - erroneous header field encountered](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/10-icmpv6-error/README.md#task-09---parameter-problem---erroneous-header-field-encountered) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
  - [x] [Task #10 - IPv6-in-IPv6 encapsulation](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/10-icmpv6-error/README.md#task-10---ipv6-in-ipv6-encapsulation) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-930370326) 
- [ ] [11-lorawan](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/11-lorawan/README.md)
  - [x] [Task #01 - LoRaWAN example](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/11-lorawan/README.md#task-01---lorawan-example) @fjmolinas (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-934505163) 
  - [x] [Task #02 - OTAA join procedure](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/11-lorawan/README.md#task-02---otaa-join-procedure) @fjmolinas (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-934505163) 
  - [ ] [Task #03 - ABP join procedure](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/11-lorawan/README.md#task-03---abp-join-procedure)
  - [x] [Task #04 - LoRaWAN device parameters persistence](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/11-lorawan/README.md#task-04---lorawan-device-parameters-persistence) @fjmolinas (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-934505163) 
  - [x] [Task #05 - GNRC LoRaWAN (Over The Air Activation)](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/11-lorawan/README.md#task-05---gnrc-lorawan-over-the-air-activation) @fjmolinas (see https://github.com/RIOT-OS/Release-Specs/issues/235#issuecomment-934505163) 
  - [ ] [Task #06 - GNRC LoRaWAN (ABP)](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/11-lorawan/README.md#task-06---gnrc-lorawan-abp)
- [ ] [99-compile-and-test-one-board](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/99-compile-and-test-one-board/README.md)
  - [ ] [Task #01 - Run tests on different hardwares](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/99-compile-and-test-one-board/README.md#task-01---run-tests-on-different-hardwares)

**Comment by fjmolinas** [comment]
LoRaWAN tests need some fixing for the automatic test to succeed, I've fixes a couple but have an issue with ABP, which I have not confirmed yet where it comes from, I should have access to hardware tomorrow to take a better look.

**Comment by fjmolinas** [comment]
@leandrolanzieri do you think you could take a look at the zephyr interopt?

**Comment by fjmolinas** [comment]
### [8. Task #05 - ICMPv6 echo between iotlab-m3 and Internet host through RIOT border router](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/08-interop/README.md#task-05---icmpv6-echo-between-iotlab-m3-and-internet-host-through-riot-border-router)

<details><summary><strong>PASSED</strong></summary>

```2a00:1450:4007:80f::2003``` is an IP of google.com. I confirmed inbound icmpv6 is working, this was running on IoT-LAB with two iotlab-m3.

```
> ifconfig
ifconfig
Iface  6  HWaddr: 67:C3  Channel: 15  Page: 0  NID: 0x76c6  PHY: O-QPSK

          Long HWaddr: 7A:94:05:FF:25:8D:E7:C3
           TX-Power: 0dBm  State: IDLE  max. Retrans.: 3  CSMA Retries: 4
          AUTOACK  ACK_REQ  CSMA  L2-PDU:102  MTU:1280  HL:64  RTR
          RTR_ADV  6LO  IPHC
          Source address length: 8
          Link type: wireless
          inet6 addr: fe80::7894:5ff:258d:e7c3  scope: link  VAL
          inet6 addr: 2001:660:3207:4d0:7894:5ff:258d:e7c3  scope: global  VAL
          inet6 group: ff02::2
          inet6 group: ff02::1
          inet6 group: ff02::1:ff8d:e7c3

          Statistics for Layer 2
            RX packets 9  bytes 480
            TX packets 9 (Multicast: 8)  bytes 424
            TX succeeded 9 errors 0
          Statistics for IPv6
            RX packets 8  bytes 608
            TX packets 9 (Multicast: 8)  bytes 608
            TX succeeded 9 errors 0

> ping6 2a00:1450:4007:80f::2003
ping6 2a00:1450:4007:80f::2003
12 bytes from 2a00:1450:4007:80f::2003: icmp_seq=1 ttl=115 rssi=-44 dBm time=153.965 ms
12 bytes from 2a00:1450:4007:80f::2003: icmp_seq=2 ttl=115 rssi=-44 dBm time=70.969 ms

--- 2a00:1450:4007:80f::2003 PING statistics ---
3 packets transmitted, 2 packets received, 33% packet loss
round-trip min/avg/max = 70.969/112.467/153.965 ms
> ping6 2a00:1450:4007:80f::2003
ping6 2a00:1450:4007:80f::2003
12 bytes from 2a00:1450:4007:80f::2003: icmp_seq=0 ttl=115 rssi=-45 dBm time=72.065 ms
12 bytes from 2a00:1450:4007:80f::2003: icmp_seq=1 ttl=115 rssi=-43 dBm time=70.874 ms
12 bytes from 2a00:1450:4007:80f::2003: icmp_seq=2 ttl=115 rssi=-44 dBm time=72.006 ms

--- 2a00:1450:4007:80f::2003 PING statistics ---
3 packets transmitted, 3 packets received, 0% packet loss
round-trip min/avg/max = 70.874/71.648/72.065 ms
```

</details>

**Comment by fjmolinas** [comment]
### [8. Task #06 - UDP between iotlab-m3 and Internet host through RIOT border router](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/08-interop/README.md#task-06---udp-between-iotlab-m3-and-internet-host-through-riot-border-router)

<details><summary><strong>PASSED</strong></summary>

On local network.

```
2021-10-05 11:47:37,611 # ifconfig
2021-10-05 11:47:37,624 # Iface  6  HWaddr: 59:4F  Channel: 26  Page: 0  NID: 0x23  PHY: O-QPSK
2021-10-05 11:47:37,624 #
2021-10-05 11:47:37,625 #           Long HWaddr: BA:F0:3B:5D:23:89:D9:4F
2021-10-05 11:47:37,627 #            TX-Power: 0dBm  State: IDLE  max. Retrans.: 3  CSMA Retries: 4
2021-10-05 11:47:37,629 #           AUTOACK  ACK_REQ  CSMA  L2-PDU:102  MTU:1280  HL:64  RTR
2021-10-05 11:47:37,630 #           RTR_ADV  6LO  IPHC
2021-10-05 11:47:37,631 #           Source address length: 8
2021-10-05 11:47:37,632 #           Link type: wireless
2021-10-05 11:47:37,634 #           inet6 addr: fe80::b8f0:3b5d:2389:d94f  scope: link  VAL
2021-10-05 11:47:37,635 #           inet6 addr: 2001:470:749d:0:b8f0:3b5d:2389:d94f  scope: global  VAL
2021-10-05 11:47:37,639 #           inet6 group: ff02::2
2021-10-05 11:47:37,640 #           inet6 group: ff02::1
2021-10-05 11:47:37,641 #           inet6 group: ff02::1:ff89:d94f
2021-10-05 11:47:37,641 #
2021-10-05 11:47:37,642 #           Statistics for Layer 2
2021-10-05 11:47:37,642 #             RX packets 3  bytes 222
2021-10-05 11:47:37,643 #             TX packets 2 (Multicast: 1)  bytes 123
2021-10-05 11:47:37,644 #             TX succeeded 2 errors 0
2021-10-05 11:47:37,644 #           Statistics for IPv6
2021-10-05 11:47:37,645 #             RX packets 2  bytes 224
2021-10-05 11:47:37,645 #             TX packets 2 (Multicast: 1)  bytes 160
2021-10-05 11:47:37,646 #             TX succeeded 2 errors 0
2021-10-05 11:47:37,646 #
udp send 2001:861:3383:9590:e5b2:c206:fff5:e42 54321 "hello"
2021-10-05 11:48:12,822 # udp send 2001:861:3383:9590:e5b2:c206:fff5:e42 54321 "hello"
2021-10-05 11:48:12,838 # Success: sent 5 byte(s) to [2001:861:3383:9590:e5b2:c206:fff5:e42]:54321
```

```
nc -u -l -6 -p 54321
hello
```
</details>

**Comment by fjmolinas** [comment]
### [8. Task #07 - UDP between iotlab-m3 and Internet host through RIOT border router (200b payload](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/08-interop/README.md#task-07---udp-between-iotlab-m3-and-internet-host-through-riot-border-router-200b-payload)

<details><summary><strong>PASSED</strong></summary>

```
→ cat data
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
ccccccccccccccccccccccccccccccccccccccccccccccccc
ddddddddddddddddddddddddddddddddddddddddddddddddd

  ~/workspace/pepper/desire  |master ↑3 ↓4 {4} U:2 ?:4 ✗|
→ cat data | nc -u 2001:470:749d:0:b8f0:3b5d:2389:d94f 54321
```

```
2021-10-05 11:47:37,611 # ifconfig
2021-10-05 11:47:37,624 # Iface  6  HWaddr: 59:4F  Channel: 26  Page: 0  NID: 0x23  PHY: O-QPSK
2021-10-05 11:47:37,624 #
2021-10-05 11:47:37,625 #           Long HWaddr: BA:F0:3B:5D:23:89:D9:4F
2021-10-05 11:47:37,627 #            TX-Power: 0dBm  State: IDLE  max. Retrans.: 3  CSMA Retries: 4
2021-10-05 11:47:37,629 #           AUTOACK  ACK_REQ  CSMA  L2-PDU:102  MTU:1280  HL:64  RTR
2021-10-05 11:47:37,630 #           RTR_ADV  6LO  IPHC
2021-10-05 11:47:37,631 #           Source address length: 8
2021-10-05 11:47:37,632 #           Link type: wireless
2021-10-05 11:47:37,634 #           inet6 addr: fe80::b8f0:3b5d:2389:d94f  scope: link  VAL
2021-10-05 11:47:37,635 #           inet6 addr: 2001:470:749d:0:b8f0:3b5d:2389:d94f  scope: global  VAL
2021-10-05 11:47:37,639 #           inet6 group: ff02::2
2021-10-05 11:47:37,640 #           inet6 group: ff02::1
2021-10-05 11:47:37,641 #           inet6 group: ff02::1:ff89:d94f
2021-10-05 11:47:37,641 #
2021-10-05 11:47:37,642 #           Statistics for Layer 2
2021-10-05 11:47:37,642 #             RX packets 3  bytes 222
2021-10-05 11:47:37,643 #             TX packets 2 (Multicast: 1)  bytes 123
2021-10-05 11:47:37,644 #             TX succeeded 2 errors 0
2021-10-05 11:47:37,644 #           Statistics for IPv6
2021-10-05 11:47:37,645 #             RX packets 2  bytes 224
2021-10-05 11:47:37,645 #             TX packets 2 (Multicast: 1)  bytes 160
2021-10-05 11:47:37,646 #             TX succeeded 2 errors 0
2021-10-05 11:47:37,646 #
> udp server start 54321
2021-10-05 11:50:51,195 # udp server start 54321
2021-10-05 11:50:51,196 # Success: started UDP server on port 54321
> 2021-10-05 11:51:19,113 # PKTDUMP: data received:
2021-10-05 11:51:19,114 # ~~ SNIP  0 - size: 199 byte, type: NETTYPE_UNDEF (0)
2021-10-05 11:51:19,115 # 00000000  61  61  61  61  61  61  61  61  61  61  61  61  61  61  61  61
2021-10-05 11:51:19,116 # 00000010  61  61  61  61  61  61  61  61  61  61  61  61  61  61  61  61
2021-10-05 11:51:19,117 # 00000020  61  61  61  61  61  61  61  61  61  61  61  61  61  61  61  61
2021-10-05 11:51:19,118 # 00000030  0A  62  62  62  62  62  62  62  62  62  62  62  62  62  62  62
2021-10-05 11:51:19,125 # 00000040  62  62  62  62  62  62  62  62  62  62  62  62  62  62  62  62
2021-10-05 11:51:19,126 # 00000050  62  62  62  62  62  62  62  62  62  62  62  62  62  62  62  62
2021-10-05 11:51:19,127 # 00000060  62  62  0A  63  63  63  63  63  63  63  63  63  63  63  63  63
2021-10-05 11:51:19,128 # 00000070  63  63  63  63  63  63  63  63  63  63  63  63  63  63  63  63
2021-10-05 11:51:19,129 # 00000080  63  63  63  63  63  63  63  63  63  63  63  63  63  63  63  63
2021-10-05 11:51:19,130 # 00000090  63  63  63  63  0A  64  64  64  64  64  64  64  64  64  64  64
2021-10-05 11:51:19,131 # 000000A0  64  64  64  64  64  64  64  64  64  64  64  64  64  64  64  64
2021-10-05 11:51:19,136 # 000000B0  64  64  64  64  64  64  64  64  64  64  64  64  64  64  64  64
2021-10-05 11:51:19,136 # 000000C0  64  64  64  64  64  64  0A
2021-10-05 11:51:19,137 # ~~ SNIP  1 - size:   8 byte, type: NETTYPE_UDP (4)
2021-10-05 11:51:19,138 #    src-port: 46820  dst-port: 54321
2021-10-05 11:51:19,138 #    length: 207  cksum: 0x035f
2021-10-05 11:51:19,139 # ~~ SNIP  2 - size:  40 byte, type: NETTYPE_IPV6 (2)
2021-10-05 11:51:19,139 # traffic class: 0x00 (ECN: 0x0, DSCP: 0x00)
2021-10-05 11:51:19,140 # flow label: 0xb4551
2021-10-05 11:51:19,140 # length: 207  next header: 17  hop limit: 63
2021-10-05 11:51:19,141 # source address: 2001:861:3383:9590:e5b2:c206:fff5:e42
2021-10-05 11:51:19,142 # destination address: 2001:470:749d:0:b8f0:3b5d:2389:d94f
2021-10-05 11:51:19,151 # ~~ SNIP  3 - size:  24 byte, type: NETTYPE_NETIF (-1)
2021-10-05 11:51:19,151 # if_pid: 6  rssi: -81  lqi: 255
2021-10-05 11:51:19,152 # flags: 0x0
2021-10-05 11:51:19,152 # src_l2addr: BA:D0:ED:FA:37:47:86:23
2021-10-05 11:51:19,153 # dst_l2addr: BA:F0:3B:5D:23:89:D9:4F
2021-10-05 11:51:19,153 # ~~ PKT    -  4 snips, total size: 271 byte
```
</details>

**Comment by leandrolanzieri** [comment]
### [Task #11 - UDP exchange between iotlab-m3 and Zephyr](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/08-interop/README.md#task-11---udp-exchange-between-iotlab-m3-and-zephyr)

I used [zephyr v2.6.0](https://github.com/zephyrproject-rtos/zephyr/releases/tag/zephyr-v2.6.0).

<details> <summary><b>PASSED</b></summary>

**RIOT node**
```
main(): This is RIOT! (Version: 2022.01-devel-HEAD)
RIOT network stack example application
All up, running the shell now

> ifconfig 6 set chan 26
ifconfig 6 set chan 26
success: set channel on interface 6 to 26

> ifconfig 6 set pan_id 0xabcd
ifconfig 6 set pan_id 0xabcd
success: set network identifier on interface 6 to 0xabcd

> udp server start 4242
udp server start 4242
Success: started UDP server on port 4242

> udp send fe80::4cea:0:e4b4:9b3b 4242 "RIOT 2021.10-RC1!"
udp send fe80::4cea:0:e4b4:9b3b 4242 "RIOT 2021.10-RC1!"
Success: sent 17 byte(s) to [fe80::4cea:0:e4b4:9b3b]:4242
PKTDUMP: data received:
~~ SNIP  0 - size:  17 byte, type: NETTYPE_UNDEF (0)
00000000  52  49  4F  54  20  32  30  32  31  2E  31  30  2D  52  43  31
00000010  21
~~ SNIP  1 - size:   8 byte, type: NETTYPE_UDP (4)
   src-port:  4242  dst-port:  4242
   length: 25  cksum: 0x383c
~~ SNIP  2 - size:  40 byte, type: NETTYPE_IPV6 (2)
traffic class: 0x00 (ECN: 0x0, DSCP: 0x00)
flow label: 0x00000
length: 25  next header: 17  hop limit: 64
source address: fe80::4cea:0:e4b4:9b3b
destination address: fe80::a405:7be5:c51e:1192
~~ SNIP  3 - size:  24 byte, type: NETTYPE_NETIF (-1)
if_pid: 6  rssi: -50  lqi: 255
flags: 0x0
src_l2addr: 4E:EA:00:00:E4:B4:9B:3B
dst_l2addr: A6:05:7B:E5:C5:1E:11:92
~~ PKT    -  4 snips, total size:  89 byte
> 
```

**Zephyr node**
```
*** Booting Zephyr OS build zephyr-v2.6.0  ***


[00:00:00.000,000] <inf> ieee802154_rf2xx: Optional instance of PORTB device activated
[00:00:00.133,000] <inf> net_echo_server_sample: Run echo server
[00:00:00.133,000] <inf> net_echo_server_sample: Network connected
[00:00:00.133,000] <inf> net_echo_server_sample: Starting...
[00:00:00.133,000] <inf> net_echo_server_sample: Waiting for UDP packets on port 4242 (IPv6)...
uart:~$ net ipv6
net ipv6
IPv6 support                              : enabled
IPv6 fragmentation support                : disabled
Multicast Listener Discovery support      : enabled
Neighbor cache support                    : enabled
Neighbor discovery support                : enabled
Duplicate address detection (DAD) support : enabled
Router advertisement RDNSS option support : enabled
6lo header compression support            : enabled
Max number of IPv6 network interfaces in the system          : 1
Max number of unicast IPv6 addresses per network interface   : 3
Max number of multicast IPv6 addresses per network interface : 4
Max number of IPv6 prefixes per network interface            : 2

IPv6 addresses for interface 1 (0x20001db8) (IEEE 802.15.4)
=========================================================
Type            State           Lifetime (sec)  Address
autoconf        preferred       infinite        fe80::4cea:0:e4b4:9b3b/128
manual          preferred       infinite        2001:db8::1/128
uart:~$ ieee802154 get_pan_id
ieee802154 get_pan_id
PAN ID 43981 (0xabcd)
uart:~$ ieee802154 get_chan
ieee802154 get_chan
Channel 26
[00:02:40.817,000] <dbg> net_echo_server_sample.process_udp: (udp6): UDP (IPv6): Received and replied with 17 bytes
[00:02:41.817,000] <err> os: ***** HARD FAULT *****
[00:02:41.817,000] <err> os: r0/a1:  0x20002fa4  r1/a2:  0x2000381b  r2/a3:  0x00000000
[00:02:41.817,000] <err> os: r3/a4:  0x00000001 r12/ip:  0x00000001 r14/lr:  0x000181f4
[00:02:41.817,000] <err> os:  xpsr:  0x00000000
[00:02:41.817,000] <err> os: Faulting instruction address (r15/pc): 0x0000000c
[00:02:41.817,000] <err> os: >>> ZEPHYR FATAL ERROR 0: CPU exception on CPU 0
[00:02:41.817,000] <err> os: Current thread: 0x20002008 (logging)
[00:02:41.885,000] <err> os: Halting system
```
</details>

**Note**: On the Zephyr node I experienced the same issue as described in https://github.com/RIOT-OS/Release-Specs/issues/219#issuecomment-829992107. I tried disabling the network logging as I suspected it was related to that but did not seem to help.

**Comment by fjmolinas** [comment]
### [8. Task #04 - ICMPv6 echo between iotlab-m3 and Internet host through Linux with 6LowPAN](https://github.com/RIOT-OS/Release-Specs/blob/d9a83ec4a3c108af6d1227049ae05b2f3d909abc/08-interop/README.md#task-04---icmpv6-echo-between-iotlab-m3-and-internet-host-through-linux-with-6lowpan)

<details><summary><strong>PASSED</strong></summary>

- rpi with 6lowpan 

```
HypriotOS/armv7: admin@riotpi-0b1263ad in ~
$ ifconfig
docker0: flags=4099<UP,BROADCAST,MULTICAST>  mtu 1500
        inet 172.17.0.1  netmask 255.255.0.0  broadcast 172.17.255.255
        ether 02:42:b0:d7:36:4b  txqueuelen 0  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

eth0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 10.42.0.140  netmask 255.255.255.0  broadcast 10.42.0.255
        inet6 fe80::ba27:ebff:fe12:63ad  prefixlen 64  scopeid 0x20<link>
        inet6 2001:861:3383:9593:ba27:ebff:fe12:63ad  prefixlen 64  scopeid 0x0<global>
        ether b8:27:eb:12:63:ad  txqueuelen 1000  (Ethernet)
        RX packets 4362  bytes 2705262 (2.5 MiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 3270  bytes 435876 (425.6 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

lo: flags=73<UP,LOOPBACK,RUNNING>  mtu 65536
        inet 127.0.0.1  netmask 255.0.0.0
        inet6 ::1  prefixlen 128  scopeid 0x10<host>
        loop  txqueuelen 1000  (Local Loopback)
        RX packets 12  bytes 668 (668.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 12  bytes 668 (668.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

lowpan0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1280
        inet6 fd23:affe::f0a7:26f0:b01:bf3b  prefixlen 64  scopeid 0x0<global>
        inet6 fe80::245d:6a0c:6f69:2ebc  prefixlen 64  scopeid 0x20<link>
        unspec 26-5D-6A-0C-6F-69-2E-BC-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 21  bytes 1632 (1.5 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 33  bytes 3262 (3.1 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

wg0: flags=209<UP,POINTOPOINT,RUNNING,NOARP>  mtu 1420
        inet 10.0.0.3  netmask 255.255.255.0  destination 10.0.0.3
        inet6 fd86:ea04:1115::3  prefixlen 64  scopeid 0x0<global>
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

wlan0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.1.53  netmask 255.255.255.0  broadcast 192.168.1.255
        inet6 fe80::ba27:ebff:fe47:36f8  prefixlen 64  scopeid 0x20<link>
        inet6 2001:861:3383:9590:b34:9662:38f7:b42d  prefixlen 64  scopeid 0x0<global>
        inet6 2001:861:3383:9590:ba27:ebff:fe47:36f8  prefixlen 64  scopeid 0x0<global>
        ether b8:27:eb:47:36:f8  txqueuelen 1000  (Ethernet)
        RX packets 615  bytes 67248 (65.6 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 60  bytes 8838 (8.6 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

wpan0: flags=195<UP,BROADCAST,RUNNING,NOARP>  mtu 123
        unspec 26-5D-6A-0C-6F-69-2E-BC-00-00-00-00-00-00-00-00  txqueuelen 300  (UNSPEC)
        RX packets 21  bytes 1010 (1010.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 60  bytes 4684 (4.5 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

HypriotOS/armv7: admin@riotpi-0b1263ad in ~
$ sudo ping6 -c 20 fd23:affe::b8f0:3b5d:2389:d94f
PING fd23:affe::b8f0:3b5d:2389:d94f(fd23:affe::b8f0:3b5d:2389:d94f) 56 data bytes
64 bytes from fd23:affe::b8f0:3b5d:2389:d94f: icmp_seq=1 ttl=255 time=14.6 ms
64 bytes from fd23:affe::b8f0:3b5d:2389:d94f: icmp_seq=2 ttl=255 time=13.2 ms
64 bytes from fd23:affe::b8f0:3b5d:2389:d94f: icmp_seq=3 ttl=255 time=13.2 ms
64 bytes from fd23:affe::b8f0:3b5d:2389:d94f: icmp_seq=4 ttl=255 time=11.3 ms
64 bytes from fd23:affe::b8f0:3b5d:2389:d94f: icmp_seq=5 ttl=255 time=12.4 ms
64 bytes from fd23:affe::b8f0:3b5d:2389:d94f: icmp_seq=7 ttl=255 time=12.3 ms
64 bytes from fd23:affe::b8f0:3b5d:2389:d94f: icmp_seq=8 ttl=255 time=12.6 ms
64 bytes from fd23:affe::b8f0:3b5d:2389:d94f: icmp_seq=9 ttl=255 time=11.3 ms
64 bytes from fd23:affe::b8f0:3b5d:2389:d94f: icmp_seq=10 ttl=255 time=12.6 ms
64 bytes from fd23:affe::b8f0:3b5d:2389:d94f: icmp_seq=11 ttl=255 time=17.5 ms
64 bytes from fd23:affe::b8f0:3b5d:2389:d94f: icmp_seq=12 ttl=255 time=13.10 ms
64 bytes from fd23:affe::b8f0:3b5d:2389:d94f: icmp_seq=13 ttl=255 time=12.7 ms
64 bytes from fd23:affe::b8f0:3b5d:2389:d94f: icmp_seq=14 ttl=255 time=14.8 ms
64 bytes from fd23:affe::b8f0:3b5d:2389:d94f: icmp_seq=15 ttl=255 time=12.7 ms
64 bytes from fd23:affe::b8f0:3b5d:2389:d94f: icmp_seq=16 ttl=255 time=12.6 ms
64 bytes from fd23:affe::b8f0:3b5d:2389:d94f: icmp_seq=17 ttl=255 time=19.4 ms
64 bytes from fd23:affe::b8f0:3b5d:2389:d94f: icmp_seq=18 ttl=255 time=13.9 ms
64 bytes from fd23:affe::b8f0:3b5d:2389:d94f: icmp_seq=19 ttl=255 time=14.2 ms
64 bytes from fd23:affe::b8f0:3b5d:2389:d94f: icmp_seq=20 ttl=255 time=13.7 ms

--- fd23:affe::b8f0:3b5d:2389:d94f ping statistics ---
20 packets transmitted, 19 received, 5% packet loss, time 102ms
rtt min/avg/max/mdev = 11.344/13.642/19.427/1.928 ms
```

- RIOT node
```
> ifconfig
2021-10-05 13:18:51,624 # ifconfig
2021-10-05 13:18:51,626 # Iface  6  HWaddr: 59:4F  Channel: 26  Page: 0  NID: 0x23  PHY: O-QPSK
2021-10-05 13:18:51,626 #
2021-10-05 13:18:51,627 #           Long HWaddr: BA:F0:3B:5D:23:89:D9:4F
2021-10-05 13:18:51,628 #            TX-Power: 0dBm  State: IDLE  max. Retrans.: 3  CSMA Retries: 4
2021-10-05 13:18:51,629 #           AUTOACK  ACK_REQ  CSMA  L2-PDU:102  MTU:1280  HL:255  RTR
2021-10-05 13:18:51,630 #           RTR_ADV  6LO  IPHC
2021-10-05 13:18:51,630 #           Source address length: 8
2021-10-05 13:18:51,631 #           Link type: wireless
2021-10-05 13:18:51,632 #           inet6 addr: fe80::b8f0:3b5d:2389:d94f  scope: link  VAL
2021-10-05 13:18:51,640 #           inet6 addr: fd23:affe::b8f0:3b5d:2389:d94f  scope: global  VAL
2021-10-05 13:18:51,641 #           inet6 group: ff02::2
2021-10-05 13:18:51,641 #           inet6 group: ff02::1
2021-10-05 13:18:51,642 #           inet6 group: ff02::1:ff89:d94f
2021-10-05 13:18:51,642 #
2021-10-05 13:18:51,643 #           Statistics for Layer 2
2021-10-05 13:18:51,643 #             RX packets 4  bytes 323
2021-10-05 13:18:51,644 #             TX packets 5 (Multicast: 2)  bytes 283
2021-10-05 13:18:51,645 #             TX succeeded 5 errors 0
2021-10-05 13:18:51,645 #           Statistics for IPv6
2021-10-05 13:18:51,646 #             RX packets 4  bytes 352
2021-10-05 13:18:51,646 #             TX packets 5 (Multicast: 2)  bytes 352
2021-10-05 13:18:51,647 #             TX succeeded 5 errors 0


> ping6 -c 20 2001:861:3383:9590:ba27:ebff:fe47:36f8
2021-10-05 13:20:42,286 # ping6 -c 20 2001:861:3383:9590:ba27:ebff:fe47:36f8
2021-10-05 13:20:42,302 # 12 bytes from 2001:861:3383:9590:ba27:ebff:fe47:36f8: icmp_seq=0 ttl=255 rssi=-78 dBm time=9.389 ms
2021-10-05 13:20:43,295 # 12 bytes from 2001:861:3383:9590:ba27:ebff:fe47:36f8: icmp_seq=1 ttl=255 rssi=-78 dBm time=10.412 ms
2021-10-05 13:20:44,303 # 12 bytes from 2001:861:3383:9590:ba27:ebff:fe47:36f8: icmp_seq=2 ttl=255 rssi=-78 dBm time=11.368 ms
2021-10-05 13:20:45,295 # 12 bytes from 2001:861:3383:9590:ba27:ebff:fe47:36f8: icmp_seq=3 ttl=255 rssi=-77 dBm time=10.345 ms
2021-10-05 13:20:46,302 # 12 bytes from 2001:861:3383:9590:ba27:ebff:fe47:36f8: icmp_seq=4 ttl=255 rssi=-78 dBm time=9.971 ms
2021-10-05 13:20:47,295 # 12 bytes from 2001:861:3383:9590:ba27:ebff:fe47:36f8: icmp_seq=5 ttl=255 rssi=-78 dBm time=9.671 ms
2021-10-05 13:20:48,302 # 12 bytes from 2001:861:3383:9590:ba27:ebff:fe47:36f8: icmp_seq=6 ttl=255 rssi=-78 dBm time=9.638 ms
2021-10-05 13:20:49,294 # 12 bytes from 2001:861:3383:9590:ba27:ebff:fe47:36f8: icmp_seq=7 ttl=255 rssi=-78 dBm time=10.021 ms
2021-10-05 13:20:50,302 # 12 bytes from 2001:861:3383:9590:ba27:ebff:fe47:36f8: icmp_seq=8 ttl=255 rssi=-78 dBm time=10.037 ms
2021-10-05 13:20:51,294 # 12 bytes from 2001:861:3383:9590:ba27:ebff:fe47:36f8: icmp_seq=9 ttl=255 rssi=-78 dBm time=9.125 ms
2021-10-05 13:20:52,302 # 12 bytes from 2001:861:3383:9590:ba27:ebff:fe47:36f8: icmp_seq=10 ttl=255 rssi=-78 dBm time=11.263 ms
2021-10-05 13:20:53,294 # 12 bytes from 2001:861:3383:9590:ba27:ebff:fe47:36f8: icmp_seq=11 ttl=255 rssi=-78 dBm time=9.446 ms
2021-10-05 13:20:54,302 # 12 bytes from 2001:861:3383:9590:ba27:ebff:fe47:36f8: icmp_seq=12 ttl=255 rssi=-78 dBm time=9.087 ms
2021-10-05 13:20:55,309 # 12 bytes from 2001:861:3383:9590:ba27:ebff:fe47:36f8: icmp_seq=13 ttl=255 rssi=-78 dBm time=12.609 ms
2021-10-05 13:20:56,302 # 12 bytes from 2001:861:3383:9590:ba27:ebff:fe47:36f8: icmp_seq=14 ttl=255 rssi=-78 dBm time=10.967 ms
2021-10-05 13:20:57,293 # 12 bytes from 2001:861:3383:9590:ba27:ebff:fe47:36f8: icmp_seq=15 ttl=255 rssi=-78 dBm time=11.634 ms
2021-10-05 13:20:58,302 # 12 bytes from 2001:861:3383:9590:ba27:ebff:fe47:36f8: icmp_seq=16 ttl=255 rssi=-77 dBm time=10.384 ms
2021-10-05 13:20:59,294 # 12 bytes from 2001:861:3383:9590:ba27:ebff:fe47:36f8: icmp_seq=17 ttl=255 rssi=-78 dBm time=8.792 ms
2021-10-05 13:21:00,301 # 12 bytes from 2001:861:3383:9590:ba27:ebff:fe47:36f8: icmp_seq=18 ttl=255 rssi=-78 dBm time=9.159 ms
2021-10-05 13:21:01,293 # 12 bytes from 2001:861:3383:9590:ba27:ebff:fe47:36f8: icmp_seq=19 ttl=255 rssi=-78 dBm time=10.954 ms
2021-10-05 13:21:01,294 #
2021-10-05 13:21:01,309 # --- 2001:861:3383:9590:ba27:ebff:fe47:36f8 PING statistics ---
2021-10-05 13:21:01,310 # 20 packets transmitted, 20 packets received, 0% packet loss
2021-10-05 13:21:01,310 # round-trip min/avg/max = 8.792/10.213/12.609 ms
```
</details>

**Comment by fjmolinas** [comment]
<h1>Test Report</h1>


<table>
  <thead>
    <tr><th></th><th>Task</th><th>Outcome</th></tr>
  </thead>
  <tbody>
  </tbody>
</table>

**Comment by benpicco** [comment]
I'll create a new RC shortly anyway once https://github.com/RIOT-OS/RIOT/pull/17005 is merged.

**Comment by miri64** [comment]
> I'll create a new RC shortly anyway once [RIOT-OS/RIOT#17005](https://github.com/RIOT-OS/RIOT/pull/17005) is merged.

_The script_ will take care of closing this one ;-).

**Comment by benpicco** [comment]
Closing in favor of #237
